.gitignore

```diff
@@ -132,4 +132,8 @@ dmypy.json
 .pytype/
 
 # Cython debug symbols
-cython_debug/
\ No newline at end of file
+cython_debug/
+
+## Custom
+.idea/
```

Comment:
Added this pattern so that your editor-specific files in `.idea/` are not version-controlled